app/models/concerns/oregon_digital/generic_metadata.rb

```diff
@@ -15,6 +15,13 @@ module GenericMetadata
       property :resource_type, predicate: ::RDF::Vocab::DC.type
       property :creator, predicate: ::RDF::Vocab::DC11.creator
       property :contributor, predicate: ::RDF::Vocab::DC11.contributor
+      property :date_created, predicate: ::RDF::Vocab::DC.created
+      property :date_modified, predicate: ::RDF::Vocab::DC.modified, multiple: false
```

Comment:
date_modified and date_uploaded should be included via Hyrax core metadata and not defined again here.

Reply:
Since we heavily modified everything, we had to strip out all the core metadata, so we added it back in here. See resource_type, creator, contributor, etc. thats where the error from the ticket #290 comes from

Reply:
Are you talking about we removed basic_metadata? resource_type, creator, etc. aren't in core_metadata

Reply:
Yeah, my bad. I was totally thinking about basic_metadata. Sometimes those seem like core to me. Momentary lapse of reasoning.